cogs/heartbeat/heartbeat.py

```diff
@@ -38,8 +39,7 @@ async def _loop(self):
                 url = await self.config.get_attr(KEY_PUSH_URL)()
                 if url:
                     LOGGER.debug("Pinging %s", url)
-                    async with aiohttp.ClientSession() as session:
-                        resp = await session.get(url)
+                    async with session.get(url) as resp:
```

Comment:
Note to self: with ClientSession above, we don't need double context managers here as seen in quickstart.